tests/integration-tests/tests/update/test_update.py

```diff
@@ -23,49 +23,48 @@
 PclusterConfig = namedtuple("PclusterConfig", ["max_queue_size", "compute_instance"])
 
 
-@pytest.mark.regions(["us-west-1"])
+@pytest.mark.regions(["eu-west-1"])
```

Comment:
No need to change but just to mention it, an alternative to the 4 annotations could have been:
`@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux", "slurm")`

Reply:
changed, thanks